community.md

```diff
@@ -0,0 +1,477 @@
+non-commercial participants.  A volunteer's motivations may be
+connected to their current job or business interests, but if they are
+ultimately participating as an individual contributor and their
+participation is likely to persist across changes of employment, then
+they have a bond with the project that goes beyond their day job and
+their motivation can be said to be non-commercial.  Every individual
+in the project usually has at least some non-commercial element to
+their participation.  Even developers who work on the project
+full-time as their salaried job generally retain their community
+status in the project if they move from employer to employer.
+
+## Sustainability, Longevity, and Lifecycle
+
+All software projects -- not just open source software projects --
+have a lifecycle.  That lifecycle may not be knowable in advance, but
+it will eventually become apparent.  As a project matures, its rate of
+user acquisition slows down, the rate at which new contributors come
+in likewise slows down, and, in the long run, the rate of new feature
+development slows as well.
+
+Projects that have an extensible design (TBD: ref "Extensibility"
+section in Adoptability module.) often have a longer lifecycle, though
+this may be correlation rather than causation: longer-lived projects
+tend to become extensible, and innovation can continue to happen in
+the plugin ecosystem even after the software's core has remained
+stable for a long time.  But, perhaps with rare exceptions, most
+software projects that last long enough eventually reache a stable
+state: either they address their problem domain so well that no
+further improvement is needed, or they are overtaken by some newer,
+nimbler competitor that was able to learn from its predecessors'
+mistakes and that does not suffer from the burdens of having an old
+code base that has accumulated [technical
+debt](https://en.wikipedia.org/wiki/Technical_debt).
+
+An organization adopting a DPG should try to ascertain, to the extent
+possible, where in its lifecycle that DPG is.  It is not necessarily a
+problem to adopt a DPG that is in the later, more static part of its
+lifecycle.  Maturity has advantages: all the easily encountered bugs
+have already been fixed, and the tool's features reflect, at least to
+some degree, the accumulated wisdom of many users' real world usage.
+But one should not expect the project's community to be eager to add
+new features or have design discussions about complicated changes.  At
+this point, the project may value stability relatively highly.  If the
+DPG needs further work to be useful for a certain purpose, then that
+work may have to happen outside the current project community.  (TBD:
+We may need to have a section explain what a fork is -- a real fork,
+that is, not a GitHub so-called "fork" :-) -- and when they are
+appropriate.)
+
+It is also not necessarily a mistake to adopt a DPG that is still in
```

Comment:
Similar to the comment above on mapping existing DPGs to the archetypes, it would be valuable in my opinion to include a "timestamped" review of current DPGs and their placement in the mature to early-stage continuum, with a commentary of the pros and cons of each stage grounded in specific DPGs

Cc: @imahgoub